syntax = "proto3";

package MxTools;

message MxpiContentString
{
    string content = 1;
}